src/Stryker.Core/Stryker.Core/TestRunners/vstest/StrykerVstestHostLauncher.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client.Interfaces;
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace Stryker.Core.TestRunners.VsTest
+{
+    public class StrykerVsTestHostLauncher : ITestHostLauncher
+    {
+        private readonly Action _callback;
+        private readonly int? _activeMutation;
+
+        public StrykerVsTestHostLauncher(Action callback, int? activeMutation)
+        {
+            _callback = callback;
+            _activeMutation = activeMutation;
+        }
+
+        public bool IsDebug => false;
+
+        public int LaunchTestHost(TestProcessStartInfo defaultTestHostStartInfo, CancellationToken cancellationToken)
+        {
+            var processInfo = new ProcessStartInfo(
+                                  defaultTestHostStartInfo.FileName,
+                                  defaultTestHostStartInfo.Arguments)
+            {
+                WorkingDirectory = defaultTestHostStartInfo.WorkingDirectory
+            };
+            processInfo.EnvironmentVariables["ActiveMutation"] = _activeMutation.ToString();
```

Comment:
This would look better:
``` csharp
var processInfo = new ProcessStartInfo(defaultTestHostStartInfo.FileName, defaultTestHostStartInfo.Arguments)
{
    WorkingDirectory = defaultTestHostStartInfo.WorkingDirectory,
    EnvironmentVariables["ActiveMutation"] = _activeMutation.ToString()
};
```

Reply:
This is not possible. EnvironmentVariables is a System.Collections.Specialized.StringDictionary. You cannot instantiate this type in that way. I would also be hesitant to overwrite the whole environment variables 'dictionary' so I would prefer not to instantiate this property. Therefore we need to add to the dictionary outside of the instantiation of ProcessStartInfo.